lib/utilities/universal_functions.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:pokedex_async_redux/utilities/constants.dart';
+
+String capitalizeFirstLetter(String name) {
+  if (name != '') {
+    return '${name[0].toUpperCase()}${name.substring(1)}';
+  } else {
+    return '';
+  }
+}
+
+Color getPokemonColor(String type) {
+  switch (type) {
+    case 'fire':
+      return kFire;
+    case 'bug':
+      return kBug;
+    case 'dark':
+      return kDark;
+    case 'electric':
+      return kElectric;
+    case 'fairy':
+      return kFairy;
+    case 'fighting':
+      return kFighting;
+    case 'dragon':
+      return kDragon;
+    case 'flying':
+      return kFlying;
+    case 'ghost':
+      return kGhost;
+    case 'grass':
+      return kGrass;
+    case 'ground':
+      return kGround;
+    case 'ice':
+      return kIce;
+    case 'normal':
+      return kNormal;
+    case 'poison':
+      return kPoison;
+    case 'psychic':
+      return kPsychic;
+    case 'rock':
+      return kRock;
+    case 'steel':
+      return kSteel;
+    case 'water':
+      return kWater;
+    case 'typeDetailsPage':
```

Comment:
what is `typeDetailsPage`?

Reply:
to get the color for this widget
![image](https://user-images.githubusercontent.com/61441278/181167682-bd331445-dbf0-4093-aae0-cd75f5844aa5.png)

Reply:
do you use the white colour? If not make this the default instead